Kiss.Bff.EndToEndTest/NieuwsEnWerkInstructies/NieuwsEnWerkInstructiesScenarios.cs

```diff
@@ -339,21 +339,36 @@ public async Task Scenario12()
     {
         await Step("Given a unique text (uuid)");
 
+        var uniqueTitle = Guid.NewGuid().ToString();
+
         await Step("Given there is exactly 1 werkinstructie with this text in the title");
 
+        var werkbericht = await Page.CreateBericht(new() { Title = uniqueTitle, BerichtType = BerichtType.Werkinstructie });
```

Comment:
The return type of this method implements `IAsyncDisposable`. If you dispose it at the end (with the `await using`) keywords, it is deleted from the application. I should have documented this a bit better. 
```suggestion
        await using var werkbericht = await Page.CreateBericht(new() { Title = uniqueTitle, BerichtType = BerichtType.Werkinstructie });
```

Reply:
addressed the comment